packages/gatsby/src/utils/eslint-rules/no-anonymous-exports-page-templates.ts

```diff
@@ -0,0 +1,65 @@
+import { Rule } from "eslint"
+import { Node, ExportDefaultDeclaration } from "estree"
+import { store } from "../../redux"
+import { isPageTemplate } from "../eslint-rules-helpers"
+
+const defs = {
```

Comment:
Do we care about anonymous classes?

Nit: To be honest, this approach feels a little over-engineered with the `defs` abstraction and `forbid` part. I would have kept this as an `if-else` or `switch` inline. But whatever :)

Reply:
I initially took it from https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-anonymous-default-export.js so that's why. I'll add the anonymous class case :)